crates/bevy_text/src/lib.rs

```diff
@@ -76,7 +76,16 @@ use bevy_sprite::SpriteSystem;
 /// When the `bevy_text` feature is enabled with the `bevy` crate, this
 /// plugin is included by default in the `DefaultPlugins`.
 #[derive(Default)]
-pub struct TextPlugin;
+pub struct TextPlugin {
+    /// Attempts to load system fonts if [true].
+    ///
+    /// Supports Windows, Linux, and macOS.
+    /// It will simply scan some predefined directories.
+    ///
+    /// For details on which directories are considered on each platform,
+    /// refer to the source code of [`fontdb::Database::load_system_fonts`](https://docs.rs/fontdb/latest/fontdb/struct.Database.html#method.load_system_fonts).
+    pub load_system_fonts: bool,
```

Comment:
As a certain "Dimchicck" on Discord suggested, this would be more accurately described as "fallback to system fonts" :)

Reply:
After giving it more thought and reading @TotalKrill's contemplations, I decided to extend the documentation for this member variable to mention its primary use case rather than renaming it.